src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -2637,57 +2638,61 @@ public static BufferedReader toBufferedReader(final Reader reader, final int siz
     }
 
     /**
-     * Gets the contents of an {@link InputStream} as a {@code byte[]}.
-     * <p>
-     * This method buffers the input internally, so there is no need to use a {@link BufferedInputStream}.
-     * </p>
+     * Reads all remaining bytes from the given {@link InputStream} into a new {@code byte[]}.
```

Comment:
What follows is just about the finer aspects of documenting methods, but it's still nice to talk about IMO:

The text can be simpler and more direct, for example, `java.nio.file.Files.read(InputStream, int)` says "Reads all the bytes from an input stream." After all, the only kinds of bytes in an input stream are the "remaining" kind, and it only makes sense to return a "new" array. So I would say: "Reads all the bytes from an input stream in a byte array".

As an aside, Java seems to have called out the "remaining" concept formally in one API name: "Iterator.forEachRemaining(...)", but that might have been to avoid overriding or shadowing the "forEach(...)".